dev/TabView/TestUI/TabViewPage.xaml

```diff
@@ -153,12 +153,14 @@
 
                         <StackPanel x:Name="FirstTabContent" AutomationProperties.Name="FirstTabContent">
                             <Button x:Name="FirstTabButton" AutomationProperties.Name="FirstTabButton" Margin="8" FontSize="20">Home Button</Button>
-                            <Button x:Name="TabViewSizingPageButton" AutomationProperties.Name="TabViewSizingPageButton"
-                                    Margin="8" Click="TabViewSizingPageButton_Click"
-                                    FontSize="20">TabView Sizing Page</Button>
-                            <Button x:Name="TabViewTabClosingBehaviorButton" AutomationProperties.Name="TabViewTabClosingBehaviorButton"
-                                    Margin="8" Click="TabViewTabClosingBehaviorButton_Click"
-                                    FontSize="20">TabView Tab Closing behavior Page</Button>
+                            <StackPanel Orientation="Horizontal">
```

Comment:
I think something went wrong with the merge, tabview test page has changes

Reply:
I did it on purpose - it helped getting the gamepad test to pass on my local runs.